examples/text_classification/download.py

```diff
@@ -0,0 +1,23 @@
+import logging
```

Comment:
Is this an example to show download function? I feel it is not relevant to text_classification things. Better to move to torch.utils docs. Or you could even put it as a separate example under examples directory